gradle.properties

```
@@ -9,15 +9,15 @@ kotlin.code.style=official
 baseIDE=idea
 
 # if you change the version of ide, also change psiViewerPluginVersion accordingly (cf https://plugins.jetbrains.com/plugin/227-psiviewer/versions)
-ideaVersion=IC-2022.1
+ideaVersion=IC-2023.1
 pycharmCommunityVersion=PC-2022.1
-psiViewerPluginVersion=221-SNAPSHOT
+psiViewerPluginVersion=231-SNAPSHOT
 
 # see https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for more information
 sinceBuild=221
```

We support only the last 3 major releases
```suggestion
sinceBuild=222
```